src/components/button/Button.tsx

```diff
@@ -1,21 +1,24 @@
 import * as React from 'react';
-import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent, Platform, AccessibilityTrait, ViewStyle } from 'react-native';
-
-const s = require('./Button.css');
+import { View, Text, TouchableOpacity, TouchableNativeFeedback, GestureResponderEvent,
```

Comment:
I would rather have everything on the same line, or split after each comma. Not a huge fan of this "random" line-break

Reply:
I agree, what about using the `max-line-length`, but add a ignore pattern for imports?

```json
{
  "max-line-length": [
    true, {
      "limit": 120,
      "ignore-pattern": "^import |^export {(.*?)}"
    }
  ]
}
```

Reply:
Sold